### DISCUSSION TITLE
model-viewer-effects not working (documentation is outdated as well)

### DISCUSSION THREAD

**Discussion by Loque18** [google/model-viewer#4356] [Q&A]
### Description
Hello, I tried adding post processing effects to one of my models, I tried with the documentation from this [link](https://modelviewer.dev/examples/postprocessing/index.html), however the links provided in the documentation were not working, it was printed in the console "error 404 not found". I had to find the links by myself on jsdeliver, I copy pasted the code provided for the pixelated and bloom effect, none of them worked, I thought maybe it was the model, so I tried with the rocket model, but the result was the same. The model is loaded, but not post processing effects are added.

#### Live Demo
[Live demo](https://loque18.github.io/mv-post-proccessing/)
[Repository](https://github.com/Loque18/mv-post-proccessing)

### Version
- model-viewer: v3.1.1

### Browser Affected

-  Chrome, version: 114.0.5735.199 (Official Build) (64-bit) (cohort: Stable) 

### OS
-  Windows 10

**Comment by elalish** [comment]
Sorry, working on getting a release out - this is a pre-release feature, so it's not in 3.1.1.

**Reply by elalish** [reply]
Also, the links are working for me?

**Reply by Loque18** [reply]
these links ?

<script type="module" src="https://cdn.jsdelivr.net/npm/@google/model-viewer/dist/model-viewer-module.min.js"></script>
  
<script type="module" src="https://cdn.jsdelivr.net/npm/@google/model-viewer-effects/dist/model-viewer-effects.min.js"></script>


they don't work for me

**Reply by elalish** [reply]
Oh, right, those ones. Sorry, should start working with the publication of the next release, which is imminent.

**Reply by elalish** [reply]
hmm, standby - still need to get the publication of the second package worked out...

**Reply by Loque18** [reply]
got it, thank you so much ! 
Can't wait for it to be released

**Reply by elalish** [reply]
Okay, we're there! https://twitter.com/modelviewer/status/1679144583312338944
